### ISSUE TITLE
Error "firstContentfulPaint": "Cannot read properties of undefined" for pages with video stream frames

### ISSUE THREAD

**Issue by RomanParaniuk** [sitespeedio/sitespeed.io#3873]
### Your question

Hi, I'm trying to test page with live-video iframe on it
I'm using docker for running test scripts 

the part of code: 
``` 
await commands.measure.start('_page');
    try {
        await commands.navigate('<URL>');
    } catch (e) {commands.error(e)}
    await commands.measure.stop();
```

### What browser did you use?

Chrome

### How to reproduce

To have such an element on the page, I believe.
Tried to test it on youtube - everything works fine

Also, I found a questions with absolutely same error
https://peaku.co/questions/47690-se-produjo-un-error-al-cambiar-a-un-iframe-con-un-reproductor-de-video-de-youtube-incrustado-[velocidad-del-sitio-io]

### Relevant log output

during test I see following INFO logs:

```
[2023-06-08 14:26:55] INFO: Take largest contentful paint screenshot
[2023-06-08 14:26:55] INFO: null
```
and in the end
```
[2023-06-08 14:28:02] ERROR: https://settings.orbitexch.com/sb_login.html generated the following errors in the coach {
  "performance": {
    "firstContentfulPaint": "Cannot read properties of undefined (reading 'startTime')",
    "longTasks": "Cannot read properties of undefined (reading 'startTime')"
  }
}
```
Could you help with it please?

**Comment by soulgalore** [comment]
Hi @RomanParaniuk thanks for creating the issue.

First the last errors:
 > "firstContentfulPaint": "Cannot read properties of undefined (reading 'startTime')",

I think that is because of a navigation that never happened. That happens in the coach script and I've seen others also having that problem, let me try to fix that ASAP, I could probably release on Monday (tomorrow).

For the video error, is there an easy way for me to reproduce that?

Best
Peter

**Comment by soulgalore** [comment]
Hi again @RomanParaniuk can you try 27.9.0, there is a fix for when Chrome misses out on FCP. But I need

> For the video error, is there an easy way for me to reproduce that?

To be able to fix that other error. Thanks.

**Comment by RomanParaniuk** [comment]
Hi @soulgalore!
Thank you a lot for such a quick response

Yes, 27.9.0 fixed issue of Coach, thank you

Regarding video, I found this site: 
https://www.bigproductions.fr/

code:
```
    await commands.measure.start('01_Main_page');
    try {
        await commands.navigate("https://www.bigproductions.fr/");
    } catch (e) {throw e}
    await commands.measure.stop();
```
Command for execution:
```
docker run --rm -v "%cd%:/sitespeed.io" sitespeedio/sitespeed.io:27.9.0 sitespeed.js -n 1 -b chrome --multi
```

Logs:

```
[2023-06-12 08:49:30] INFO: Running tests using Chrome - 1 iteration(s)
[2023-06-12 08:49:30] INFO: Start to measure 01_Main_page
[2023-06-12 08:49:31] INFO: Navigating to url https://www.bigproductions.fr/ iteration 1
[2023-06-12 08:49:39] INFO: Take after page complete check screenshot
[2023-06-12 08:49:40] INFO: Take cumulative layout shift screenshot
[2023-06-12 08:49:41] INFO: Take largest contentful paint screenshot
[2023-06-12 08:49:41] INFO: null
[2023-06-12 08:49:43] INFO: Use the visual metrics portable script
[2023-06-12 08:49:43] INFO: Get visual metrics from the video
[2023-06-12 08:49:48] INFO: https://www.bigproductions.fr/ 16 requests, TTFB: 157ms, firstPaint: 445ms, firstVisualChange: 474ms, FCP: 445ms, DOMContentLoaded: 522ms, CLS: 0.0016, TBT: 0ms, CPUBenchmark: 55ms, Load: 670ms, speedIndex: 753ms, visualComplete85: 474ms, lastVisualChange: 2.92s
```

So it does not return any errors now (same as on my site) but this null is still present.
I'm not sure if it's critical issue which need to be fixed. But anyway.

Thank you!

**Comment by soulgalore** [comment]
Thank you @RomanParaniuk for taking the time to find another site with the issue, I'll have a go tonight and reproduce/fix it.

**Comment by soulgalore** [comment]
I checked the code and the "null" comes from when we try to make a red rectangle where the largest contentful paint is and the Chrome API seems to only report parts of it, I'll check if I can fix that (at least change that strange log message).

**Comment by soulgalore** [comment]
This should be fixed now in 27.9.1.

**Comment by RomanParaniuk** [comment]
Thank you a lo @soulgalore !
I tested - it fixed in 27.9.1
Closing the ticket
